rest-api-spec/src/main/resources/rest-api-spec/test/indices.data_stream/10_basic.yml

```diff
@@ -28,7 +24,7 @@ setup:
             mappings:
               properties:
                 '@timestamp':
-                  type: date
+                  type: date_nanos
```

Comment:
This is changed to exemplify a custom `@timestamp` mapping

Reply:
Maybe in the `Create data stream` yaml test, also check that mapping for the backing index of each data streams to have the correct mapping?